package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "unifiedpush-node-sender",
-  "version": "0.15.0",
+  "version": "0.16.0",
```

Comment:
i don't think this should be updated in the PR.  it will be updated during the release